avm/res/digital-twins/digital-twins-instance/tests/e2e/pe/main.test.bicep

```diff
@@ -0,0 +1,87 @@
+targetScope = 'subscription'
+
+metadata name = 'Using large parameter set'
+metadata description = 'This instance deploys the module with most of its features enabled.'
```

Comment:
Careful, if you deploy the PE also in a seperate deployment test, please update the metadata to call it out

Reply:
updated